docs/zh_cn/get_started.md

```diff
@@ -1,169 +1,275 @@
+**步骤 2.** 按照[官方说明](https://pytorch.org/get-started/locally/)安装 PyTorch，例如：
 
-### 使用 CPU 训练
+在 GPU 平台上：
 
 ```shell
-export CUDA_VISIBLE_DEVICES=-1
-python tools/train.py ${CONFIG_FILE}
+conda install pytorch torchvision -c pytorch
 ```
 
-**注意**: 我们不推荐用户使用 CPU 进行训练， 因为 CPU 的训练速度很慢，一些算法仅支持分布式训练, 例如 `SyncBN`，该方法需要分布式进行训练，我们支持这个功能是为了方便用户在没有 GPU 的机器上进行调试。
-
-### 使用 单张/多张 显卡训练
+在 CPU 平台上：
 
 ```shell
-sh tools/dist_train.sh ${CONFIG_FILE} ${GPUS} --work-dir ${YOUR_WORK_DIR} [optional arguments]
+conda install pytorch torchvision cpuonly -c pytorch
 ```
 
-可选参数:
+## 安装
+
+我们建议用户遵循我们的最佳实践来安装 MMSelfSup。 但是，整个过程是高度可定制的。 有关详细信息，请参阅[自定义安装](#customize-installation)部分。
 
-- `--resume-from ${CHECKPOINT_FILE}`: 从某个 checkpoint 处继续训练.
-- `--deterministic`: 开启 "deterministic" 模式, 虽然开启会使得训练速度降低，但是会保证结果可复现。
+### 最佳实践
 
-例如:
+**步骤 0.** 使用 [MIM](https://github.com/open-mmlab/mim) 安装 [MMEngine](https://github.com/open-mmlab/mmengine) 和 [MMCV](https://github.com/open-mmlab/mmcv)。
 
 ```shell
-# checkpoints and logs saved in WORK_DIR=work_dirs/selfsup/odc/odc_resnet50_8xb64-steplr-440e_in1k/
-sh tools/dist_train.sh configs/selfsup/odc/odc_resnet50_8xb64-steplr-440e_in1k.py 8 --work_dir work_dirs/selfsup/odc/odc_resnet50_8xb64-steplr-440e_in1k/
+pip install -U openmim
+mim install mmengine
+mim install 'mmcv>=2.0.0rc1'
 ```
 
-**注意**: 在训练过程中, checkpoints 和 logs 被保存在同一目录层级下.
+**步骤 1.** 安装 MMSelfSup。
 
-此外, 如果您在一个被 [slurm](https://slurm.schedmd.com/) 管理的集群中训练， 您可以使用以下的脚本开展训练:
+根据您的需要，我们支持两种安装方式：
+
+- [从源代码安装（推荐）](#从源代码安装): 您想开发自己的自监督任务或基于 MMSelfSup 框架的新功能，例如，添加新的数据集或模型。 您可以使用我们提供的所有工具。
+- [作为 Python 包安装](#作为-python-包安装): 您只想在项目中调用 MMSelfSup 的 API 或导入 MMSelfSup 的模块。
+
+#### 从源代码安装
+
+在这种情况下，从源代码安装 MMSelfSup：
 
 ```shell
-GPUS_PER_NODE=${GPUS_PER_NODE} GPUS=${GPUS} SRUN_ARGS=${SRUN_ARGS} sh tools/slurm_train.sh ${PARTITION} ${JOB_NAME} ${CONFIG_FILE} ${YOUR_WORK_DIR} [optional arguments]
+git clone https://github.com/open-mmlab/mmselfsup.git
+cd mmselfsup
+git checkout 1.x
+pip install -v -e .
+# "-v" 表示详细，或更多输出
+# "-e" 表示以可编辑模式安装项目，
+# 因此，对代码所做的任何本地修改都将生效，无需重新安装。
 ```
 
-例如:
+或者，如果您想为 MMSelfSup 做出贡献或体验其正在实验中的功能，请查看 `dev-1.x` 分支：
 
 ```shell
-GPUS_PER_NODE=8 GPUS=8 sh tools/slurm_train.sh Dummy Test_job configs/selfsup/odc/odc_resnet50_8xb64-steplr-440e_in1k.py work_dirs/selfsup/odc/odc_resnet50_8xb64-steplr-440e_in1k/
+git checkout dev-1.x
 ```
 
-### 使用多台机器训练
+#### 作为 Python 包安装
 
-如果您想使用由 ethernet 连接起来的多台机器， 您可以使用以下命令:
-
-在第一台机器上:
+直接用 pip 安装：
 
 ```shell
-NNODES=2 NODE_RANK=0 PORT=$MASTER_PORT MASTER_ADDR=$MASTER_ADDR sh tools/dist_train.sh $CONFIG $GPUS
+pip install 'mmselfsup>=1.0.0rc0'
 ```
 
-在第二台机器上:
+### 验证安装
 
-```shell
-NNODES=2 NODE_RANK=1 PORT=$MASTER_PORT MASTER_ADDR=$MASTER_ADDR sh tools/dist_train.sh $CONFIG $GPUS
+要验证是否正确安装了 MMSelfSup，可以运行以下命令：
+
+```python
+import mmselfsup
+print(mmselfsup.__version__)
+# 示例输出：1.0.0rc0 或更新版本
+```
+
+我们还提供以下示例代码来初始化模型并推断演示图像：
```

Comment:
This python snippet is for 0.x version, I think we could remove this part, please also remove this snippet in English version.